modules/Administration/Search/MVC/View.php

```diff
@@ -77,8 +78,12 @@ public function preDisplay()
         if (!isset($sugar_config['search']) || !$sugar_config['search']) {
             LoggerManager::getLogger()->warn('Configuration does not contains default search settings.');
         }
-        $search = isset($sugar_config['search']) ? $sugar_config['search'] : null;
+        $search = $sugar_config['search'] ?? null;
```

Comment:
Hi @Dillon-Brown can we use `??` in this branch? do we still have to keep `5.6` support?

Reply:
Develop will be PHP 7.3 minimum (Currently 7.2 until the composer update PR get merged).